index.qmd

```diff
@@ -16,108 +16,96 @@ library(shiny)
 library(ggplot2)
 library(bslib)
 
-ui <- fluidPage(
+ui <- page_fillable(
   theme = bs_theme(bg = "#fcfefe", fg = "#595959"),
-  mainPanel(
-    fluidRow(
-      column(
-        width = 4,
-        align = "center",
-        sliderInput(
-          "beta_1",
-          label = "Predictor 1 slope",
-          min = -4.0,
-          max =  4.0,
-          step = 0.5,
-          value = 1
-        )
-      ),
-      column(
-        width = 4,
-        align = "center",
-        sliderInput(
-          "beta_2",
-          label = "Predictor 2 slope",
-          min = -4.0,
-          max =  4.0,
-          step = 0.5,
-          value = 1
-        )
-      ),
-      column(
-        width = 4,
-        align = "center",
-        sliderInput(
-          "beta_int",
-          label = "Interaction slope",
-          min = -2.0,
-          max =  2.0,
-          step = 0.25,
-          value = 0.5
-        )
-      )
-    ), # row
-    fluidRow(
-      column(width =  1),
-      column(width = 10.0, align = "center", plotOutput('contours')),
-      column(width =  1)
-    ) # row
-  )  # main
-) # ui page
+  padding = "1rem",
+  layout_columns(
+    fill = FALSE,
+    col_widths = breakpoints(xs = c(-2, 8, -2), sm = 4),
+    sliderInput(
+      "beta_1",
+      label = "Predictor 1 slope",
+      min = -4.0,
+      max = 4.0,
+      step = 0.5,
+      value = 1,
+      ticks = FALSE
+    ),
+    sliderInput(
+      "beta_2",
+      label = "Predictor 2 slope",
+      min = -4.0,
+      max = 4.0,
+      step = 0.5,
+      value = 1,
+      ticks = FALSE
+    ),
+    sliderInput(
+      "beta_int",
+      label = "Interaction slope",
+      min = -2.0,
+      max = 2.0,
+      step = 0.25,
+      value = 0.5,
+      ticks = FALSE
+    )
+  ),
+  as_fill_carrier(plotOutput("contours"))
```

Comment:
After the next release of bslib, this could just be

```suggestion
  plotOutput("contours")
```

Without the `as_fill_carrier()`, there's a bug in `bslib::page_fillable()` that prevents the plot from filling available space. If you can live with that until the next release (this month), you can take the suggestion above.